### PR TITLE
Removal of _ formatting, replacing with * formatting

### PR DESCRIPTION
**Pull request by KellieHawks**
Replacing the "_" bold formatting to the more favorable "*" formatting.

### CODE REVIEW

File: _docs/_developer_guide/platform_wide/analytics_overview.md [hunk 1]
```diff
@@ -187,22 +187,22 @@ Numeric attributes have a wide variety of use-cases. Incrementing number custom
...
-| Check if the numeric attribute __is more than__ a __numver__| __MORE THAN__ | __NUMVER__ |
-| Check if the numeric attribute __is less than__ a __number__| __LESS THAN__ | __NUMBER__ |
-| Check if the numeric attribute __is exactly__ a __number__| __EXACTLY__ | __NUMBER__ |
-| Check if the numeric attribute __does not equal__ a __numver__| __DOES NOT EQUAL__ | __NUMBER__ |
-| Check if the numeric attribute __exists__ on a user's profile | __EXISTS__ | __N/A__ |
-| Check if the numeric attribute __does not exist__ on a user's profile | __DOES NOT EXIST__ | __N/A__ |
+| Check if the numeric attribute **is more than** a **numver**| **MORE THAN** | **NUMVER** |
+| Check if the numeric attribute **is less than** a **number**| **LESS THAN** | **NUMBER** |
+| Check if the numeric attribute **is exactly** a **number**| **EXACTLY** | **NUMBER** |
+| Check if the numeric attribute **does not equal** a **numver**| **DOES NOT EQUAL** | **NUMBER** |
```
Comment (KellieHawks) [inline _docs/_developer_guide/platform_wide/analytics_overview.md:193]:
```suggestion
| Check if the numeric attribute **does not equal** a **number**| **DOES NOT EQUAL** | **NUMBER** |
```

File: _docs/_developer_guide/platform_wide/app_group_configuration.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ If users will simply have to update the app and it’s not a new app being relea
...
 
-The best way to go about this if you want to separate the old vs. new app would be to create a new app within the same app group. This way, you can effectively target only users on the new version by selecting that app as you create your segments. If you did want to message users who are still on the old version, you can use [Liquid to select the old app and filter the previous app version](https://www.youtube.com/watch?v=Dv__RAUwamA).
+The best way to go about this if you want to separate the old vs. new app would be to create a new app within the same app group. This way, you can effectively target only users on the new version by selecting that app as you create your segments. If you did want to message users who are still on the old version, you can use [Liquid to select the old app and filter the previous app version](https://www.youtube.com/watch?v=Dv**RAUwamA).
```
Comment (KellieHawks) [inline _docs/_developer_guide/platform_wide/app_group_configuration.md:59]:
```suggestion
The best way to go about this if you want to separate the old vs. new app would be to create a new app within the same app group. This way, you can effectively target only users on the new version by selecting that app as you create your segments. If you did want to message users who are still on the old version, you can use [Liquid to select the old app and filter the previous app version](https://www.youtube.com/watch?v=Dv__RAUwamA).
```

File: _docs/_hidden/other/support_contact.md [hunk 3]
```diff
@@ -1172,7 +1172,7 @@ $( document ).ready(function() {
            noResults({createElement}) {
              return createElement("div", {
                dangerouslySetInnerHTML: {
-                 __html: '<div class="no_results">No results were found with your current search. Try to change the search query.</div>',
+                 **html: '<div class="no_results">No results were found with your current search. Try to change the search query.</div>',
```
Comment (KellieHawks) [inline _docs/_hidden/other/support_contact.md:1175]:
```suggestion
                 __html: '<div class="no_results">No results were found with your current search. Try to change the search query.</div>',
```

File: _docs/_hidden/other/support_contact.md [hunk 4]
```diff
@@ -1223,7 +1223,7 @@ $( document ).ready(function() {
                 "</div></div></a>";
             return createElement("div", {
               dangerouslySetInnerHTML: {
-                __html: resulttemplate,
+                **html: resulttemplate,
```
Comment (KellieHawks) [inline _docs/_hidden/other/support_contact.md:1226]:
```suggestion
                __html: resulttemplate,
```

File: _docs/_home/styling_test_page.md [hunk 5]
```diff
@@ -335,7 +335,7 @@ from engine import RunForrestRun
 """Test code for syntax highlighting!"""
 
 class Foo:
-	def __init__(self, var):
+	def **init**(self, var):
```
Comment (KellieHawks) [inline _docs/_home/styling_test_page.md:338]:
```suggestion
	def __init__(self, var):
```

File: _docs/_partners/data_and_infrastructure_agility/support/pypestream.md [hunk 6]
```diff
@@ -49,7 +49,7 @@ import os
 NAME = '{ CUSTOMER NAME }'
 BOTS = []
 CSV_BOTS = ['{ SOLUTION NAME }']
-PATH = os.path.dirname(__file__)
+PATH = os.path.dirname(**file**)
```
Comment (KellieHawks) [inline _docs/_partners/data_and_infrastructure_agility/support/pypestream.md:52]:
```suggestion
PATH = os.path.dirname(__file__)
```

File: _docs/_partners/data_and_infrastructure_agility/workflow_automation/zapier.md [hunk 7]
```diff
@@ -49,10 +49,10 @@ First, choose **POST** as the webhook action type. Next, make sure to fill out t
 
 - **Webhook URL**: `https://rest.iad-01.braze.com/campaigns/trigger/send`
 - **Payload Type** : JSON
-- **Data** : `trigger_properties__name`, `api_key`, and `campaign_id`
+- **Data** : `trigger_properties**name`, `api_key`, and `campaign_id`
 These data fields are key-value pairs that will be used for the data portion of the request.
 
-![The data key-value pairs for this example are "app_group_id" set as "your-app-group-id", "campaign_id" set as "your-campaign-id", and "trigger_properties__name" set as "Post Name".][10]
+![The data key-value pairs for this example are "app_group_id" set as "your-app-group-id", "campaign_id" set as "your-campaign-id", and "trigger_properties**name" set as "Post Name".][10]
```
Comment (KellieHawks) [inline _docs/_partners/data_and_infrastructure_agility/workflow_automation/zapier.md:55]:
```suggestion
![The data key-value pairs for this example are "app_group_id" set as "your-app-group-id", "campaign_id" set as "your-campaign-id", and "trigger_properties__name" set as "Post Name".][10]
```

File: _docs/_partners/data_and_infrastructure_agility/workflow_automation/zapier.md [hunk 8]
```diff
@@ -49,10 +49,10 @@ First, choose **POST** as the webhook action type. Next, make sure to fill out t
 
 - **Webhook URL**: `https://rest.iad-01.braze.com/campaigns/trigger/send`
 - **Payload Type** : JSON
-- **Data** : `trigger_properties__name`, `api_key`, and `campaign_id`
+- **Data** : `trigger_properties**name`, `api_key`, and `campaign_id`
```
Comment (KellieHawks) [inline _docs/_partners/data_and_infrastructure_agility/workflow_automation/zapier.md:52]:
```suggestion
- **Data** : `trigger_properties__name`, `api_key`, and `campaign_id`
```

File: _docs/_partners/message_orchestration/channel_extensions/loyalty/talonone.md [hunk 9]
```diff
@@ -127,7 +127,7 @@ Use Braze [Connected Content]({{site.baseurl}}/user_guide/personalization_and_dy
 {% alert tip %}
 You can access the coupon code with {% raw %} `{{result.value}}` {% endraw %} as shown above, which will return the generated value similar to `44D4-U4PL`.
 
-You can also access the entire response from Talon.One by accessing {% raw %} `{{result}}` {% endraw %} directly, which will look similar to `{"id"=>1548040, "value"=>"44D4-U4PL", "__http_status_code__"=>200}`.
+You can also access the entire response from Talon.One by accessing {% raw %} `{{result}}` {% endraw %} directly, which will look similar to `{"id"=>1548040, "value"=>"44D4-U4PL", "**http_status_code**"=>200}`.
```
Comment (KellieHawks) [inline _docs/_partners/message_orchestration/channel_extensions/loyalty/talonone.md:130]:
```suggestion
You can also access the entire response from Talon.One by accessing {% raw %} `{{result}}` {% endraw %} directly, which will look similar to `{"id"=>1548040, "value"=>"44D4-U4PL", "__http_status_code__"=>200}`.
```

File: _docs/_partners/message_personalization/location/radar.md [hunk 10]
```diff
@@ -137,7 +137,7 @@ Bringing our use-case together, here is what the syntax of the campaign would lo
 {% raw %}
 ```
 {% connected_content https://api.radar.io/v1/search/places?radius=2000&near={{${most_recent_location}.latitude}},{{${most_recent_location}.longitude}}&chains=target,walmart&limit=5 :method get :headers {"Authorization": "<yourRadarPublishableKey>"} :content_type application/json :save nearbyplaces %}
-{% if nearbyplaces.__http_status_code__ != 200 %}
+{% if nearbyplaces.**http_status_code** != 200 %}
```
Comment (KellieHawks) [inline _docs/_partners/message_personalization/location/radar.md:140]:
```suggestion
{% if nearbyplaces.__http_status_code__ != 200 %}
```

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags.md [hunk 11]
```diff
@@ -177,11 +177,11 @@ This is a simple example of how iteration tags can be used in Braze's message co
 
 ## HTTP status codes {#http-personalization}
 
-You can utilize the HTTP status from a [Connected Content][38] call by first saving it as a local variable and then using the `__http_status_code__` key. For example:
+You can utilize the HTTP status from a [Connected Content][38] call by first saving it as a local variable and then using the `**http_status_code**` key. For example:
 
 ```html
 {% connected_content https://example.com/api/endpoint :save connected %}
-{% if connected.__http_status_code__ != 200 %}
+{% if connected.**http_status_code** != 200 %}
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags.md:184]:
```suggestion
{% if connected.__http_status_code__ != 200 %}
```

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags.md [hunk 12]
```diff
@@ -177,11 +177,11 @@ This is a simple example of how iteration tags can be used in Braze's message co
 
 ## HTTP status codes {#http-personalization}
 
-You can utilize the HTTP status from a [Connected Content][38] call by first saving it as a local variable and then using the `__http_status_code__` key. For example:
+You can utilize the HTTP status from a [Connected Content][38] call by first saving it as a local variable and then using the `**http_status_code**` key. For example:
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags.md:180]:
```suggestion
You can utilize the HTTP status from a [Connected Content][38] call by first saving it as a local variable and then using the `__http_status_code__` key. For example:
```

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/local_connected_content_variables.md [hunk 13]
```diff
@@ -122,12 +122,12 @@ If you want to provide your own JSON body, you can write it inline if there are
...
 ## HTTP status codes
 
-You can utilize the HTTP status from a Connected Content call by first saving it as a local variable and then using the `__http_status_code__` key. For example:
+You can utilize the HTTP status from a Connected Content call by first saving it as a local variable and then using the `**http_status_code**` key. For example:
 
 {% raw %}
 ```js
 {% connected_content https://example.com/api/endpoint :save result %}
-{% if result.__http_status_code__ != 200 %}
+{% if result.**http_status_code** != 200 %}
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/connected_content/local_connected_content_variables.md:130]:
```suggestion
{% if result.__http_status_code__ != 200 %}
```

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/local_connected_content_variables.md [hunk 14]
```diff
@@ -122,12 +122,12 @@ If you want to provide your own JSON body, you can write it inline if there are
 
 ## HTTP status codes
 
-You can utilize the HTTP status from a Connected Content call by first saving it as a local variable and then using the `__http_status_code__` key. For example:
+You can utilize the HTTP status from a Connected Content call by first saving it as a local variable and then using the `**http_status_code**` key. For example:
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/connected_content/local_connected_content_variables.md:125]:
```suggestion
You can utilize the HTTP status from a Connected Content call by first saving it as a local variable and then using the `__http_status_code__` key. For example:
```

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/implementation_guide.md [hunk 15]
```diff
@@ -78,13 +78,13 @@ extension ContentCardData: Equatable {
...
 {% subtab Objective-C %}
-__ContentCardable Protocol__<br>
+**ContentCardable Protocol**<br>
 A `ContentCardData` object that represents the `ABKContentCard` data along with a `ContentCardClassType` enum, an initializer used to instantiate custom objects with `ABKContentCard` metadata.
 ```objc
 @protocol ContentCardable <NSObject>
  
 @property (nonatomic, strong) ContentCardData *contentCardData;
-- (instancetype __nullable)initWithMetaData:(NSDictionary *)metaData
+- (instancetype **nullable)initWithMetaData:(NSDictionary *)metaData
```
Comment (bre-fitzgerald) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/implementation_guide.md:87]: @KellieHawks  should this be omitted from the change?

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/implementation_guide.md [hunk 16]
```diff
@@ -525,19 +525,19 @@ Visit the [following section](#logging-impressions-clicks-and-dismissals) to get
 
 After extending your custom objects to function as Content Cards, logging valuable metrics like impressions, clicks, and dismissals is quick and simple. This can be done through the use of a `ContentCardable` protocol that references and provides data to a helper file to be logged by the Braze SDK.
 
-#### __Implementation components__<br><br>
+#### **Implementation components**<br><br>
```
Comment (bre-fitzgerald) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/implementation_guide.md:528]:
```suggestion
#### Implementation components<br><br>
```

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/implementation_guide.md [hunk 17]
```diff
@@ -83,11 +83,11 @@ While building out your slide-up in-app message, you may notice you aren't able
 
 Visit the [SlideFromBottomViewController](https://github.com/braze-inc/braze-growth-shares-ios-demo-app/blob/master/Braze-Demo/ViewController/In-App-Messages/SlideFromBottomViewController.swift) to get started.
 
-#### __Adding additional behavior to our default UI__<br><br>
+#### **Adding additional behavior to our default UI**<br><br>
```
Comment (bre-fitzgerald) [inline _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/implementation_guide.md:86]:
```suggestion
#### Adding additional behavior to our default UI<br><br>
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Looks good! Few small things for your review :)

**Review by bre-fitzgerald — DISMISSED** [review]

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**